Mathlib/CategoryTheory/Category/RelCat.lean

```diff
@@ -21,81 +21,80 @@ By flipping the arguments to a relation, we construct an equivalence `opEquivale
 `RelCat` and its opposite.
 -/
 
+open Rel
+
 namespace CategoryTheory
 
 universe u
 
--- This file is about Lean 3 declaration "Rel".
-
 /-- A type synonym for `Type u`, which carries the category instance for which
     morphisms are binary relations. -/
 def RelCat :=
   Type u
 
-instance RelCat.inhabited : Inhabited RelCat := by unfold RelCat; infer_instance
-
-/-- The category of types with binary relations as morphisms. -/
-instance rel : LargeCategory RelCat where
-  Hom X Y := X → Y → Prop
-  id _ x y := x = y
-  comp f g x z := ∃ y, f x y ∧ g y z
-
+namespace RelCat
+variable {X Y Z : RelCat.{u}}
 
+instance inhabited : Inhabited RelCat := by unfold RelCat; infer_instance
 
-namespace RelCat
+/-- The morphisms in the relation category are relations. -/
+structure Hom (X Y : RelCat.{u}) : Type u where
```

Comment:
This looks like an independent refactor that we should do first

Reply:
#25593